gnxi/gnxi.go

```diff
@@ -11,20 +11,20 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 
-	gnmi "github.com/Azure/sonic-telemetry/gnmi_server"
-	testcert "github.com/Azure/sonic-telemetry/testdata/tls"
+	gnmi "github.com/sonic-net/sonic-gnmi/gnmi_server"
```

Comment:
>github.com/sonic-net [](http://example.com/codeflow?start=7&length=20)

Possible to define a string constant for the prefix?

Reply:
It seems that Go import does not support string constant.